tests/sandbox/sandbox.sh

```diff
@@ -13,14 +13,6 @@ VERSION_FILE="${PATH_HERE}"/../share/VERSION_LONG
 is_package_specified=false
 is_js_bundle_specified=false
 
-# Install tree
-PLATFORM_FILE="${PATH_HERE}"/../share/PLATFORM
-# If not there, try locally
-if [ ! -f "${VERSION_FILE}" ]; then
-    PLATFORM_FILE=PLATFORM
-fi
-platform=$(<"${PLATFORM_FILE}")
-
```

Comment:
When platform was a build-time decision, we created a file named `PLATFORM` and installed it, so that that `sandbox.sh` could automatically infer what type of binaries it was surrounded by. This doesn't exist anymore - it's surrounded by platform-agnostic binaries. `--enclave-platform` can still be passed explicitly to `sandbox.sh`/`start_network.py`/similar wrappers, and is inserted for all of our `e2e` tests, but doesn't come from file checks.